package.json

```diff
@@ -15,6 +15,8 @@
     "babel-plugin-transform-es2015-modules-commonjs": "6.24.0",
     "babel-preset-es2015": "6.24.0",
     "babel-preset-es2015-loose": "8.0.0",
+    "chai": "3.5.0",
```

Comment:
I added several packages that I think will help with promises, coverage and source mapping. Gonna keep them around for now, if you don't feel too strongly about this.